pype/hosts/blender/plugin.py

```diff
@@ -20,6 +20,36 @@ def asset_name(
     return name
 
 
+def asset_namespace(
+    asset: str, subset: str
+) -> str:
+    """Return a unique namespace based on the asset name."""
+    avalon_containers = [
+        c for c in bpy.data.collections 
+        if c.name == 'AVALON_CONTAINERS'
+    ]
+    loaded_assets = []
+    for c in avalon_containers:
+        loaded_assets.extend(c.children)
+    if avalon_containers is None:
+        return "1"
+    collections_names = [
+        c.name for c in loaded_assets 
+    ]
+    count = 1
+    name = f"{asset_name(asset, subset, str(count))}_CON"
```

Comment:
Why is `name` calculated here and in the loop if not used? If should be used, then do not call that on each iteration but call it only once after the loop.

Reply:
`name` is used in the while condition! It is checking if `name` is in `collections_names`!